static/js/impala/persona_creation.js

```diff
@@ -280,17 +280,13 @@
             updatePersona();
         }).trigger('change');
 
-        // Check for native `input[type=color]` support (i.e., WebKit).
```

Comment:
We don't use `type=color` inputs, so this is useless. We might want to in the future though.